scenarios/examples/inference-scheduling.sh

```diff
@@ -0,0 +1,94 @@
+# INFERENCE SCHEDULING WELL LIT PATH
+# Based on https://github.com/llm-d-incubation/llm-d-infra/tree/main/quickstart/examples/inference-scheduling
+# Removed pod monitoring; can be added using LLMDBENCH_VLLM_MODELSERVICE_EXTRA_POD_CONFIG
+# Removed extra volumes metrics-volume and torch-compile-volume; they are not needed for this model and tested hardware.
+# Use LLMDBENCH_VLLM_MODELSERVICE_EXTRA_VOLUME_MOUNTS and LLMDBENCH_VLLM_MODELSERVICE_EXTRA_VOLUMES to add them if needed.
+
+# IMPORTANT NOTE
+# All parameters not defined here or exported externally will be the default values found in setup/env.sh
+# Many commonly defined values were left blank (default) so that this scenario is applicable to as many environments as possible.
+
+# Model parameters
+# export LLMDBENCH_DEPLOY_MODEL_LIST="Qwen/Qwen3-0.6B"
+# export LLMDBENCH_DEPLOY_MODEL_LIST="facebook/opt-125m"
+export LLMDBENCH_DEPLOY_MODEL_LIST="meta-llama/Llama-3.1-8B-Instruct"
+#export LLMDBENCH_DEPLOY_MODEL_LIST="meta-llama/Llama-3.1-70B-Instruct"
+export LLMDBENCH_VLLM_COMMON_PVC_MODEL_CACHE_SIZE=1Ti
+
+
+# Workload parameters
+export LLMDBENCH_HARNESS_EXPERIMENT_PROFILE=shared_prefix_synthetic.yaml
+export LLMDBENCH_HARNESS_NAME=inference-perf
+
+# Routing configuration (via gaie)
+#export LLMDBENCH_VLLM_MODELSERVICE_GAIE_PLUGINS_CONFIGFILE="plugins-v2.yaml" (default is default-plugins.yaml)
```

Comment:
I don't see the experiment file as part of this PR. It should be updated to replace `LLMDBENCH_VLLM_MODELSERVICE_GAIE_PRESETS` with `LLMDBENCH_VLLM_MODELSERVICE_GAIE_PLUGINS_CONFIGFILE`.